src/styles/block/card.scss

```diff
@@ -0,0 +1,55 @@
+
+.card {
+  box-sizing: border-box;
```

Comment:
it s better to apply this property for all elements (use * selector)

when all elements have the same `box-sizing` value, their behavior becomes more predictable